kura/org.eclipse.kura.rest.identity.provider/src/main/java/org/eclipse/kura/internal/rest/identity/provider/IdentityService.java

```diff
@@ -77,18 +81,24 @@ public void createUser(UserDTO user) throws KuraException {
         }
     }
 
-    public void deleteUser(String userName) {
-        this.userAdminHelper.deleteUser(userName);
+    public void deleteUser(String userName) throws WebApplicationException {
```

Comment:
I dont' like the identityservice throw `WebApplicationException` . Throw the KuraException and then catch and convert it in WebApplicationException